components/organization/Repository.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="box-backdrop">
+    <div class="box">
+      <header class="box-header">
+        <slot name="header">
+          <!--{{repository.name}} -->
+          Client
+        </slot>
+      </header>
+
+      <section class="box-description">
+        <slot name="description">
+          <!-- {{repository.description}} -->
+          UNDER CONSTR Uddd fffdd ddddd dddd d dd ddC T ION
+        </slot>
+      </section>
+
+      <section class="box-languages">
+        <slot name="languages">
+          <!--   
+            {{repoditory.languange}}
+            -->
+          Javascript | Vue | HTML
+        </slot>
+      </section>
+
+      <footer class="box-footer">
+        <slot name="footer">
+          <img width="20px" height="20px" src="..\..\static\fork.png" /> 5
```

Comment:
You didn't upload this img and also if you upload images in the `static` folder you can do that:

```suggestion
          <img width="20px" height="20px" src="/fork.png" /> 5
```